src/LinuxTracing/LibunwindstackUnwinder.h

```diff
@@ -20,16 +20,26 @@
 
 namespace orbit_linux_tracing {
 
+struct LibunwindstackResult {
+  std::vector<unwindstack::FrameData> frames;
+  // If "true", Libunwindstack successfully unwound the callstack, but the actual stack has more
+  // frames than what was given as "max_frames", so this callstack is incomplete.
```

Comment:
Came here to copy this `struct`. Just a heads up that this comment is outdated.

Reply:
Thanks